extensions/sql-migration/src/wizard/targetSelectionPage.ts

```diff
@@ -263,6 +263,8 @@ export class TargetSelectionPage extends MigrationWizardPage {
 			if (selectedIndex > -1) {
 				this.migrationStateModel._azureAccount.properties.tenants = [this.migrationStateModel.getTenant(selectedIndex)];
 			}
+			await this.populateSubscriptionDropdown();
+			await this.populateLocationDropdown();
```

Comment:
this should not be done here.  locations should be based on locations available by the selected subscription

Reply:
Done, only refreshes subscription now when tenant is changed